data/core/editor/help.cfg

```diff
@@ -165,6 +165,44 @@ This turns any base terrain into a village, providing income and healing." + "
 Fake Shroud looks like an unexplored area, even in scenarios that have shroud disabled and even when the player’s units can see the hex."
 [/topic]
 
+[topic]
+    id=editor_terrain_elevation
+    title= _ "Terrain Elevation Graphics"
+    text= _ "editor^<img>src='icons/terrain/terrain_group_elevation_30.png~SCALE(60,60)' align=left box=yes</img> - A cosmetic elevation effect that looks a little different in the editor than in the game due to visual aids in the editor; all the overlays described here are found in the terrain palette’s “special” group." + "
```

Comment:
I think it's better to put the images in a non-translatable string, for example as in the `editor_tool_village` section.